config/routes.rb

```diff
@@ -17,6 +17,7 @@
       to: 'v1/sessions#new',
       constraints: ->(request) { V1::SessionsController::REDIRECT_URLS.include?(request.path_parameters[:type]) }
   get '/v1/sessions/ssoe_logout', to: 'v1/sessions#ssoe_slo_callback'
+  get 'v1/sessions/tracker', to: 'v1/sessions#tracker'
```

Comment:
Nitpick, for consistency with the other routes (even though further down in the file there are routes without it. 🤷 ):
```suggestion
  get '/v1/sessions/tracker', to: 'v1/sessions#tracker'
```